Documentation/dotnet-mage/README.md

```diff
@@ -56,3 +56,5 @@ Suppose that we have copied project output to a sub-folder `files` and our .NET
 * Create deployment manifest
 
 `mage.net -new Deployment -Install true -pub "My Publisher" -v 1.0.0.1 -AppManifest files\MyApp.manifest -t MyApp.application`
+
+### To update an existing application, repeat steps above.
```

Comment:
Actually, the steps don't have to be repeated - there is an "-update" command that should be used. This is explained in the document I linked above: https://docs.microsoft.com/en-us/dotnet/framework/tools/mage-exe-manifest-generation-and-editing-tool

I suggest updating the comment to direct users to that document for more usage scenarios, like preparing an update or signing the application.

Reply:
@NikolaMilosavljevic Thanks, I will update.